src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/prison/PrisonApiClient.kt

```diff
@@ -29,6 +29,29 @@ class PrisonApiClient(@Qualifier("oauthPrisonClient") val prisonerSearchApiWebCl
     return booking
   }
 
+  fun resetHdcChecks(bookingId: Long): Boolean {
+    val response = prisonerSearchApiWebClient
+      .delete()
+      .uri("/offender-sentences/booking/{bookingId}/home-detention-curfews/latest/checks-passed", bookingId)
+      .accept(MediaType.APPLICATION_JSON)
+      .exchangeToMono {
+        Mono.just(it)
+      }
+      .block()
+
+    return if (response.statusCode().is2xxSuccessful) {
+      log.info("Successfully reset licence for booking $bookingId, status:${response.statusCode()}")
+      true
+    } else {
+      log.warn(
+        "Failed to reset licence for booking $bookingId, status:${response.statusCode()}, uri: ${response.request().uri}, body: ${
+          response.toEntity(String::class.java).block()
+        }",
+      )
+      false
```

Comment:
an unsuccessful response is logged but doesn't kill the job.. 409 here would mean the record is currently locked